### ISSUE TITLE
Only first `style` is processed

### ISSUE THREAD

**Issue by simonwjackson** [vue-styleguidist/vue-styleguidist#95]
```html
<div style="height: 300px; width: 300px;">
```

only height gets processed. Everything else is forgotten.

**Comment by rafaesc** [comment]
Hi @simonwjackson,

Please, give more information about this issue and how can replicate it.

Regards

**Comment by simonwjackson** [comment]
```html
<docs>
```vue
<template>
  <div style="width: 300px; height: 900px'>
      dummy text
  </div>
</template>
</docs>
```

Here is an example of a documentation tag inside a component.

when i open styleguidist, the `div` will have a width of 300px but not a height of 900px

**Comment by rafaesc** [comment]
Yes, I think that this issue could be fixed with this PR

https://github.com/vue-styleguidist/vue-styleguidist/pull/93

**Comment by mrjones2014** [comment]
@rafaesc I actually don't know that PR #93  would fix that issue. I wrote that pull request to process the `<style scoped>` tag in the Single File Component format, but I didn't write any code that would process the `style` attribute of individual elements.

**Comment by mrjones2014** [comment]
However, you could simply extract these styles to a class and put the class in a `<style scoped>` tag.

**Comment by simonwjackson** [comment]
@mrjones2014 i was trying to use inline styles as a workaround since `<style scoped>` is broken.

**Comment by mrjones2014** [comment]
@simonwjackson I've submitted a PR #93 which should add support for `<style scoped>`. if you don't actually _need_ inline styles, #93 should fix your issue whenever it's merged, I'd guess within the next few days.

**Comment by rafaesc** [comment]
@simonwjackson It's fixed on version 1.6.1.
